docs/app/Examples/collections/Menu/Types/MenuExampleVerticalDropdown.js

```diff
@@ -13,7 +13,7 @@ export default class MenuExampleVerticalDropdown extends Component {
       <Menu secondary vertical>
         <Menu.Item name='account' active={activeItem === 'account'} onClick={this.handleItemClick} />
         <Menu.Item name='settings' active={activeItem === 'settings'} onClick={this.handleItemClick} />
-        <Dropdown as={Menu.Item} text='Display Options'>
+        <Dropdown text='Display Options' pointing='left' className='link item'>
```

Comment:
Ah ok, sorry for being dense 😄  I was just thinking about the doc change not the longer term, more complete, fix for the code. I'll see what I can do.